go.mod

```diff
@@ -12,9 +12,10 @@ require (
 	github.com/orijtech/prometheus-go-metrics-exporter v0.0.6
 	github.com/osrg/gobgp v0.0.0-20210801043420-9e48a36ed97c
 	github.com/pkg/errors v0.9.1
+	github.com/pkg/profile v1.6.0
 	github.com/projectcalico/api v0.0.0-20220129171738-695e97a25204 // v3.22.0
 	github.com/projectcalico/calico v0.0.0-20220128225346-a86e41d021c4 // v3.22.0
-	github.com/prometheus/common v0.10.0
+	github.com/prometheus/client_golang v1.7.1
```

Comment:
Are those two really needed ?

Reply:
no no you're right that was a mistake related to some local files I have